internal/builders/container/README.md

```diff
@@ -0,0 +1,255 @@
+        id: build
+        with:
+          push: true
+          tags: ${{ steps.meta.outputs.tags }}
+          labels: ${{ steps.meta.outputs.labels }}
+
+      - name: Output image and digest
+        id: digest
+        env:
+          DIGEST: ${{ steps.build.outputs.digest }}
+        run: |
+          # NOTE: We need to use the image and digest in order to make sure
+          # that the image we attest has not been modified.
+          # NOTE: The digest output from docker/build-push-action is of the
+          # form "sha256:<digest>"
+          image_name="${IMAGE_REGISTRY}/${IMAGE_NAME}@${DIGEST}"
+          echo "::set-output name=image::$image_name"
+
+  # This step calls the container workflow to generate provenance and push it to
+  # the container registry.
+  provenance:
+    needs: [build]
+    permissions:
+      actions: read # for detecting the Github Actions environment.
+      id-token: write # for creating OCID tokens for signing.
+      packages: write # for uploading attestations.
+    if: startsWith(github.ref, 'refs/tags/')
+    uses: slsa-framework/slsa-github-generator/.github/workflows/generator_container_slsa3.yml@main
+    with:
+      image: ${{ needs.build.outputs.image-and-digest }}
+      registry-username: ${{ github.actor }}
+      # TODO(https://github.com/slsa-framework/slsa-github-generator/issues/492): Remove after GA release.
+      compile-generator: true
+    secrets:
+      registry-password: ${{ secrets.GITHUB_TOKEN }}
+```
+
+### Supported Triggers
+
+The following [GitHub trigger events](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows) are fully supported and tested:
+
+- `schedule`
+- `push` (including new tags)
+- `release`
+- Manual run via `workflow_dispatch`
+
+However, in practice, most triggers should work with the exception of
+`pull_request`. If you would like support for `pull_request`, please tell us
+about your use case on [issue #358](https://github.com/slsa-framework/slsa-github-generator/issues/358). If
+you have an issue in all other triggers please submit a [new
+issue](https://github.com/slsa-framework/slsa-github-generator/issues/new/choose).
+
+### Workflow Inputs
+
+The [container workflow](https://github.com/slsa-framework/slsa-github-generator/blob/main/.github/workflows/generator_container_slsa3.yml) accepts the following inputs:
+
+Inputs:
+
+| Name                | Required | Description                                                                                         |
+| ------------------- | -------- | --------------------------------------------------------------------------------------------------- |
+| `image`             | yes      | The OCI image name. This must not include a tag or digest.                                          |
+| `digest`            | yes      | The OCI image digest. The image digest of the form '<algorithm>:<digest>' (e.g. 'sha256:abcdef...') |
```

Comment:
the README's example workflow uses `image` to pass both to image + digest, but the doc here says otherwise.

Reply:
Yeah, I forgot to update the example after I added the digest input

Reply:
Done.